self-api/src/main/java/com/selfxdsd/api/ApiToken.java

```diff
@@ -0,0 +1,29 @@
+package com.selfxdsd.api;
+
+import java.time.LocalDateTime;
+
+/**
+ * API Token.
+ */
+public interface ApiToken {
+    /**
+     * The name of token.
+     *
+     * @return String.
+     */
+    String name();
+
+    /**
+     * Secret key.
+     *
+     * @return Byte array.
+     */
+    byte[] secret();
+
+    /**
+     * The expiration date.
+     *
+     * @return LocalDateTime.
+     */
+    LocalDateTime expiration();
+}
```

Comment:
@andreoss add a ``@todo`` in this class, to implement method ``User :: owner()``, we will need that as well :D

Reply:
@amihaiemil Done.